internal/test/framework/framework.go

```diff
@@ -294,6 +295,28 @@ func (f *TestFramework) GetNode(externalIP string) (*v1.Node, error) {
 	return matchedNode, nil
 }
 
+// ApplyHybridOverlayPatch will enable the hybrid overlay on the cluster
+func (f *TestFramework) ApplyHybridOverlayPatch() error {
+	jsonPatch := []byte(`{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"hybridOverlayConfig":` +
+		`{"hybridClusterNetwork":[{"cidr":"10.132.0.0/14","hostPrefix":23}]}}}}}`)
+
+	err := f.patchClusterNetworkOperator(jsonPatch)
+	if err != nil {
+		return fmt.Errorf("could not apply patch %s to cluster network: %s", string(jsonPatch), err)
+	}
+
+	return nil
+}
+
+// patchClusterNetworkOperator patches the "network.operator cluster" resource, with the provided JSON patch
+func (f *TestFramework) patchClusterNetworkOperator(patch []byte) error {
```

Comment:
Should we use the same logic for `wmcb` and remove patching cno from shell script?

Reply:
Sure but that should be done in a separate PR

Reply:
Yeah, mention it somewhere in the code, so that we won't forget.

Reply:
@ravisantoshgudimetla it makes more sense to open a ticket if we want this work done. I've created https://issues.redhat.com/browse/WINC-224, wasnt sure what to put in the description so please fill it out :smile: